### ISSUE TITLE
Issues with post install script

### ISSUE THREAD

**Issue by RichardLewin87** [nodegit/nodegit#1129]
I am currently trying to install nodegit v0.15.1 via npm but it is failing to complete the post-install script with the following error:

>Error: The specified module could not be found.
\\?\C:\Code\...\temp\electron-projects\node_modules\nodegit\build\Release\nodegit.node
    at Error (native)
    at Object.Module._extensions..node (module.js:568:18)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Code\..\electron-projects\node_modules\nodegit\dist\nodegit.js:11:12)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
at ChildProcess.exithandler (child_process.js:202:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:492:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node C:\\Code\\...\\temp\\electron-projects\\node_modules\\nodegit\\dist\\nodegit.js' }

Tracking up a few lines though and this file is reported to have downloaded ok:

> [nodegit] Running install script
[nodegit] Success: "C:\Code\...\temp\electron-projects\node_modules\nodegit\build\Release\nodegit.node" is installed via remote
[nodegit] Completed installation successfully.

I have confirmed on that it is indeed on the disk. Therefore I am baffled as toi why it is erroring out as it is.

Incase it is important I am running the following node and npm versions:

Node: v6.1.0
NPM: v3.8.6

Please let me know if any more information is needed and thanks in advance.

**Comment by maxkorp** [comment]
This is fixed as of 0.16.0. It's because it's compiled for electron, not node, so when we try to load it up in node it dies. The check is bypassed when being built from electron in newer versions.

**Comment by RichardLewin87** [comment]
Hi @maxkorp. I did as you suggested and it did solve the issue I outlined so thank you for the help with that. However, I am now encountering a new problem. When it is downloading the pre-built electron it is attempting to download Electron v0.37.8 that, I assume from the message I get, is not available (it is very old version of Electron now so I understand this being the case). Error in console I get:

> [nodegit] Running pre-install script
[nodegit] Running install script
node-pre-gyp
ERR! Tried to download: https://nodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.16.0-electron-v0.37-win32-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for nodegit@0.16.0 and electron@0.37.8 (electron-v0.37 ABI) (falling back to source compile with node-gyp)
gyp

However, in all my package.json files I am referencing the very latest version of Electron, v1.4.0. Indeed if I go back to nodegit 0.15.1 it then downloads Electron v1.4.0 without a problem, but not when I switch back to 0.16.0.

It seems that something is locking it to Electron v0.37.8 when I use nodegit v0.16.0. I can't find anything in my code away from nodegit that is locking it down though.
